internal/photoprism/openstreetmap.go

```diff
@@ -106,11 +106,11 @@ func (m *MediaFile) Location() (*models.Location, error) {
 	location.LocCountryCode = strings.TrimSpace(openstreetmapLocation.Address.CountryCode)
 	location.LocDisplayName = strings.TrimSpace(openstreetmapLocation.DisplayName)
 
-	locationCategory := strings.TrimSpace(strings.ReplaceAll(openstreetmapLocation.Category, "_", " "))
+	locationCategory := strings.TrimSpace(strings.Replace(openstreetmapLocation.Category, "_", " ", -1))
 	location.LocCategory = locationCategory
 
 	if openstreetmapLocation.Type != "yes" && openstreetmapLocation.Type != "unclassified" {
-		locationType := strings.TrimSpace(strings.ReplaceAll(openstreetmapLocation.Type, "_", " "))
```

Comment:
ReplaceAll was added in 1.12. Anyone using Go version <1.12 will get build failure for develop

Reply:
Good to know! Shouldn't be an issue since we provide docker images containing all dependencies.